phpunit

```diff
@@ -53,6 +53,7 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
     chdir("phpunit-$PHPUNIT_VERSION");
     passthru("$COMPOSER remove --no-update phpspec/prophecy");
     passthru("$COMPOSER remove --no-update symfony/yaml");
+    passthru("$COMPOSER require --no-update phpunit/phpunit-mock-objects \"~3.1.0\"");
```

Comment:
it would be great to fix it in phpunit (using a conflict rule in ``phpunit/phpunit-mock-objects`` maybe)